crates/sparrow-syntax/src/syntax/fenl_type.rs

```diff
@@ -101,12 +103,14 @@ pub enum TypeConstraint {
     Number,
     /// Any signed numeric type.
     Signed,
-    /// Any floating point numeric ytpe.
+    /// Any floating point numeric type.
     Float,
     /// Any time delta.
     TimeDelta,
     /// Any ordered type. This includes numbers and timestamps.
     Ordered,
+    /// Error type
+    Error,
```

Comment:
This is for erroring out not that we have an Error Type in the fenl right?

Reply:
This is for collecting multiple errors -- we need a way of recording "we've reported a failure, so don't report more issues with this".